src/test/groovy/io/seqera/migtool/MigToolTest.groovy

```diff
@@ -41,6 +40,27 @@ class MigToolTest extends Specification {
         conn?.close()
     }
 
+    def cleanup() {
```

Comment:
Is a cleanup phase necessary? Being an H2 in-memory database, isn't a fresh one being created on every new connection?

Reply:
not each test, only each run so when I was running all tests at once - It was conflicting. I had to run each test separately. It is a bit ugly, but I didn't find a nicer way at the time.

Reply:
> not each test, only each run

I'm not sure if I'm understanding correctly, but `cleanup` runs at the end of every test method while `cleanupSpec` runs at the end of all test methods.

> It is a bit ugly, but I didn't find a nicer way at the time

In the test that was already there, I've noticed that the database is not deleted once the connection closes [on purpose](https://github.com/seqeralabs/migtool/blob/6d253dda72fd7bfef74426432e4d1c467feb5f34/src/test/groovy/io/seqera/migtool/MigToolTest.groovy#L18) by using `DB_CLOSE_DELAY=-1`, because the tool might open several connections.

In order to avoid the cleanup phase, we could use the same connection string in the rest of the tests with a different database name each time (`jdbc:h2:mem:test[1,2,3];DB_CLOSE_DELAY=-1`).